### PR TITLE
[Aqicn.org] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/pull/17741

### CODE REVIEW

File: src/chrome/content/rules/Aqicn.org.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		- waqi		(m)
+
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
```
Comment (pipboy96) [inline src/chrome/content/rules/Aqicn.org.xml]:
```xml
<!--
	Other related ruleset:
		- Waqi.info.xml

	Mismatched:
		- www
		- feedback
		- ios
		- waqi
-->
```

**Comment by pipboy96** [comment]
More subdomains exist:
```
api.aqicn.org
d.aqicn.org
ftp.aqicn.org
geo.aqicn.org
jp1.aqicn.org
jp2.aqicn.org
jp3.aqicn.org
mail.aqicn.org
mapi.aqicn.org
mapi3.aqicn.org
mapidroid.aqicn.org
mapidroid2.aqicn.org
scs.aqicn.org
sg.aqicn.org
sg1.aqicn.org
stat.aqicn.org
static.aqicn.org
sw2.aqicn.org
sw3.aqicn.org
tiles2.aqicn.org
wgt.aqicn.org
widget.aqicn.org
winwgt.aqicn.org
```

**Comment by youdly** [comment]
Updated comment.

**Comment by J0WI** [comment]
#17740 and #17741 depend on each other.

**Comment by youdly** [comment]
Removed "related ruleset" comment.

**Pull request merged**